ipwb/indexer.py

```diff
@@ -230,8 +230,9 @@ def getCDXJLinesFromFile(warcPath, **encCompOpts):
             try:
                 ctype = hdrs.get_header('Content-Type')
                 if ctype and ctype.lower().startswith('text/html'):
-                    title = BeautifulSoup(payload, 'html.parser').title.text
-                    title = ' '.join(title.split()) or None
+                    title = BeautifulSoup(payload, 'html.parser').title
+                    if title is not None:
+                        title = ' '.join(title.text.split())
```

Comment:
Please change this line to `title = ' '.join(title.split()) or None` otherwise there will be cases where it will return an empty string. Doing this will make sure either we have a non-empty title or return `None`.

Reply:
@ibnesayeed Done in 95aba90. It was unclear why you put the `or None` logic, hence the removal.